dist/images/ovnkube.sh

```diff
@@ -290,13 +293,14 @@ check_ovn_daemonset_version() {
 get_ovn_db_vars() {
   ovn_nbdb_str=""
   ovn_sbdb_str=""
-  for i in ${!ovn_db_hosts[@]}; do
-    if [[ ${i} -ne 0 ]]; then
+  for i in "${ovn_db_hosts[@]}"; do
```

Comment:
rename “i” to "ip" instead to make it clear? "i" usually refers to an interger.

Reply:
sure